src/main/java/com/nexus/sion/feature/member/command/application/controller/UserCertificateHistoryController.java

```diff
@@ -22,7 +22,8 @@ public class UserCertificateHistoryController {
   // 개발자 자격증 등록
   @PostMapping
   public ResponseEntity<ApiResponse<Void>> registerUserCertificate(
-      @PathVariable String employeeId, @RequestBody @Valid UserCertificateHistoryRequest request) {
+      @PathVariable String employeeId,
```

Comment:
```suggestion
      @AuthenticationPrincipal UserDetails userDetails,
```
userDetails.getUserName() <- 이렇게 가져오는게 더 안전해욥